src/components/Calculator.js

```diff
@@ -0,0 +1,120 @@
+/* eslint-disable */
+import React, { Component } from 'react';
+
+import calculate from './logic/calculate';
+import operate from './logic/operate';
```

Comment:
- [ ] `operate.js `is already being used by `calculate.js`, so there's no need to import it